### PR TITLE
Tlv2 comments

### PR DESCRIPTION
**Pull request by husobee**

### CODE REVIEW

File: server/server_test.go [hunk 1]
```diff
@@ -237,6 +239,78 @@ func (suite *ServerTestSuite) TestNewIssueRedeemV2() {
...
+
+	err := suite.srv.createV3Issuer(issuer)
+	suite.Require().NoError(err)
+
+	time.Sleep(30 * time.Second)
+
+	err = suite.srv.rotateIssuersV3()
+	suite.Require().NoError(err)
+
+	//issuerKey, err := suite.srv.GetLatestIssuer(issuer.IssuerType, issuer.IssuerCohort)
```
Comment (evq) [inline server/server_test.go:263]: ?

File: server/tokens.go [hunk 2]
```diff
@@ -259,8 +260,9 @@ func (c *Server) blindedTokenRedeemHandlerV3(w http.ResponseWriter, r *http.Requ
 			}
 
 		}
-		response = blindedTokenRedeemResponse{verifiedCohort}
+		response = blindedTokenRedeemResponse{issuer.IssuerCohort}
```
Comment (evq) [inline server/tokens.go:263]: do the cohorts actually work as expected for v3 issuers? it seems like we only ever retrieve one issuer

File: server/cron.go [hunk 3]
```diff
@@ -27,5 +27,14 @@ func (c *Server) SetupCronTasks() {
 	}); err != nil {
 		panic(err)
 	}
+	//if _, err := cron.AddFunc(cadence, func() {
```
[the remark this answers is not in the document]
Comment (husobee) [reply server/cron.go:30]: adding this back in now.

File: server/tokens.go (hunk 2 above)
Comment (husobee) [reply server/tokens.go:268]: removing the concept of issuers for time aware creds.

Comment (husobee) [reply server/tokens.go:268]: https://github.com/brave-intl/challenge-bypass-server/issues/256

**Comment by husobee** [comment]
https://github.com/brave/security/issues/738 the challenge bypass updates for this security review for tlv2 issuers and redemptions and key rotations.

File: server/db.go [hunk 4]
```diff
@@ -669,6 +718,21 @@ func (c *Server) rotateIssuersV3() error {
 	return nil
 }
 
+// deleteIssuerKeys deletes issuers keys that have ended more than the duration ago.
+func (c *Server) deleteIssuerKeys(duration string) (int64, error) {
```
Comment (evq) [inline server/db.go:722]: is this safe for non-v3 rotated keys / use cases?

Comment (evq) [reply server/db.go:722]: cc @Sneagan

Comment (husobee) [reply server/db.go:722]: https://github.com/brave-intl/challenge-bypass-server/pull/252/commits/111fb321527d10c2e11b29e05e8890bfbb56565b

fixed to apply only to v3 issuer keys.

**Review by evq — APPROVED** [review]

**Pull request merged**